app/src/common/shared/org/mozilla/vrbrowser/browser/Accounts.kt

```diff
@@ -329,6 +329,10 @@ class Accounts constructor(val context: Context) {
             SyncEngine.History -> {
                 GleanMetricsService.FxA.historySyncStatus(value)
             }
+            //TODO
+//            SyncEngine.Passwords -> {
+//                GleanMetricsService.FxA.loginsSyncStatus(value)
+//            }
```

Comment:
Does this mean saved passwords are not synced back to FxA from FxR?

Reply:
It was just a comment to remember to add Telemetry, I've removed it. I'll open a task to add Telemetry for both Addons and Autofill as the review process takes a while and we are already out of time.